LinkedList/LinkedList.cs

```diff
@@ -47,6 +58,7 @@ public class LinkedList<T> : ILinkedList<T>
 
         public void Add(T item)
         {
+            onAddItem?.Invoke("Add element to LinkedList");
```

Comment:
Calling event should be moved to separted method
Naming: "OnEvent", e.g. OnAdd, OnRemove, OnEmpty